### PR TITLE
Fix Resiliency test cases

### PR DESCRIPTION
**Pull request by VijithaEkanayake**
## Purpose

Enable resiliency test cases.

### CODE REVIEW

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public void setup() {
      */
     @Test
     public void testCircuitBreaker() {
-        int[] expectedStatusCodes = new int[] { 200, 200, 502, -1, -1, 200, 200, 200 };
```
Comment (afkham) [inline tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java]: Please put a comment on what this -1 means

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public void setup() {
...
+        int[] expectedStatusCodes = new int[] { 200, 200, 502, 0, 0, 200, 200, 200 };
```
Comment (afkham) [inline tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java]: Please put a comment on what this 0 means

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java [hunk 3]
```diff
@@ -56,13 +56,11 @@ public void testCircuitBreaker() {
         BRefValueArray errs = (BRefValueArray) returnVals[1];
 
         for (int i = 0; i < responses.size(); i++) {
-            BStruct res = (BStruct) responses.get(i);
             long statusCode;
-
-            if (res != null) {
+            if (i != 3 && i != 4) {
```
Comment (afkham) [inline tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java]: Please put comments about all these magic numbers in the code

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java (hunk 1 above)
Comment (VijithaEkanayake) [reply tests/ballerina-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java]: These status codes haven't been used anymore.

**Pull request merged**